uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaJMXNavigatorTest.java

```diff
@@ -0,0 +1,244 @@
+		try {
+			console.terminateConsole();
+			new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		} catch (CoreLayerException ex) {
+			log.warn("Cannot terminate a console. Perhaps there is no active console.");
+		}
+
+		log.info("Deleting all projects");
+		ProjectFactory.deleteAllProjects();
+	}
+
+	/**
+	 * <p>
+	 * Create a new Fuse integration Project (CBR, blueprint, the latest Camel version).
+	 * </p>
+	 * <ul>
+	 * <li>Run as Local Camel Context (to create a launch configuration)</li>
+	 * <li>Stop the running Local Camel Context</li>
+	 * <li>Go to Run --> Run Configurations... --> Local Camel Context --> select created launch configuration</li>
+	 * <li>Switch to JRE tab and add new VM argument: <i>-javaagent:path/to/jolokia-jvm-1.3.7-agent.jar</i></li>
+	 * <li>Apply changes and hit Run button</li>
+	 * <li>Notice: you should get something like Jolokia: Agent started with URL <i>http://127.0.0.1:8778/jolokia/</i>
+	 * in Console View</li>
+	 * <li>Create a new connection in JMX Navigator</li>
+	 * <li>New connection... --> <i>Jolokia Connection</i></li>
+	 * <li>Provide the name you want</li>
+	 * <li>Use Jolokia URL from Console View (by default <i>http://127.0.0.1:8778/jolokia</i>)</li>
+	 * <li>Select <i>Do NOT verify SSL Certificates</i></li>
+	 * <li>Click Finish</li>
+	 * <li>Expand User-Defined Connections --> double-click on the connection to activate it</li>
+	 * <li>Try to work with it (accessing nodes, perform available operations, ...)</li>
+	 * </ul>
+	 */
+	@Test
+	public void testJolokiaAcccessingCamelRoute() {
+
+		CamelProject project = new CamelProject(PROJECT_NAME);
+
+		log.info("Run the Fuse project as Local Camel Context");
+		project.runCamelContext();
+
+		log.info("Try to stop running local camel context via terminate a console.");
+		ConsoleView console = new ConsoleView();
+		console.open();
+		try {
+			console.terminateConsole();
+			new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		} catch (CoreLayerException ex) {
+			log.warn("Cannot terminate a console. Perhaps there is no active console.");
+		}
+
+		log.info(
+				" Try to open 'Run -> Run Configurations...' and select created configuration for Local Camel Context");
+		RunConfigurationsDialog run = new RunConfigurationsDialog();
+		project.update();
+		project.selectProjectItem("src/main/resources", "OSGI-INF", "blueprint", PROJECT_CAMEL_CONTEXT);
+		new ContextMenuItem("Run As", "Run Configurations...").select();
+		new WaitUntil(new ShellIsAvailable("Run Configurations"));
+		run.select(new LocalCamelContextLaunchConfiguration(),
+				"Run " + PROJECT_NAME + " (blueprint.xml) as Local CamelContext");
+
+		log.info("Try to set VM argument for Jolokia jvm agent and run local camel context");
+		LocalCamelContextLaunchConfigurationJRETab jre = new LocalCamelContextLaunchConfigurationJRETab();
+		jre.activate();
+		jre.setTextVMArgumentsTXT("-javaagent:" + System.getProperty("jolokia"));
+		jre.clickApplyBTN();
+		run.run();
```

Comment:
It looks like, here is the root cause. You do not wait until _Run as Local Camel Context_ is finished.